<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>892</width>
    <height>648</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QTableView" name="RankingTable">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>20</y>
      <width>151</width>
      <height>551</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>60</x>
      <y>30</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Ranking</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Database_select_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>20</y>
      <width>121</width>
      <height>22</height>
     </rect>
    </property>
    <property name="text">
     <string>select dataset</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Preprocessing_select_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>70</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>select preprocessing</string>
    </property>
   </widget>
   <widget class="QToolButton" name="Algorithm_select_button">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>100</y>
      <width>121</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>select algorithm</string>
    </property>
   </widget>
   <widget class="QLabel" name="dataset_path_label">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>50</y>
      <width>531</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
   </widget>
   <widget class="QLabel" name="dataset_filename_label">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>20</y>
      <width>111</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>No File</string>
    </property>
   </widget>
   <widget class="QScrollArea" name="dataset_scrollarea">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>130</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>159</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="x_test_scrollarea">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>300</y>
      <width>391</width>
      <height>161</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_2">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>159</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QScrollArea" name="x_test_scrollarea_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>500</y>
      <width>391</width>
      <height>31</height>
     </rect>
    </property>
    <property name="widgetResizable">
     <bool>true</bool>
    </property>
    <widget class="QWidget" name="scrollAreaWidgetContents_3">
     <property name="geometry">
      <rect>
       <x>0</x>
       <y>0</y>
       <width>389</width>
       <height>29</height>
      </rect>
     </property>
    </widget>
   </widget>
   <widget class="QLabel" name="label_2">
    <property name="geometry">
     <rect>
      <x>170</x>
      <y>480</y>
      <width>101</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>Selected x_test</string>
    </property>
   </widget>
   <widget class="QProgressBar" name="progressBar">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>90</y>
      <width>291</width>
      <height>23</height>
     </rect>
    </property>
    <property name="value">
     <number>24</number>
    </property>
   </widget>
   <widget class="QGraphicsView" name="loss_accuracy_graph">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>130</y>
      <width>271</width>
      <height>171</height>
     </rect>
    </property>
   </widget>
   <widget class="QGraphicsView" name="confusion_metrix">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>310</y>
      <width>271</width>
      <height>181</height>
     </rect>
    </property>
   </widget>
   <widget class="QTextBrowser" name="accuracy_f1score">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>510</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLabel" name="label_5">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>490</y>
      <width>121</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>accuracy / f1_score</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_6">
    <property name="geometry">
     <rect>
      <x>600</x>
      <y>530</y>
      <width>60</width>
      <height>16</height>
     </rect>
    </property>
    <property name="text">
     <string>y_test</string>
    </property>
   </widget>
   <widget class="QTextBrowser" name="accuracy_f1score_2">
    <property name="geometry">
     <rect>
      <x>590</x>
      <y>550</y>
      <width>271</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QPushButton" name="train_button">
    <property name="geometry">
     <rect>
      <x>450</x>
      <y>90</y>
      <width>113</width>
      <height>32</height>
     </rect>
    </property>
    <property name="text">
     <string>start train</string>
    </property>
   </widget>
   <widget class="QComboBox" name="hidden_layer_select">
    <property name="geometry">
     <rect>
      <x>580</x>
      <y>30</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>relu</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>sigmoid</string>
     </property>
    </item>
   </widget>
   <widget class="QComboBox" name="output_layer_select">
    <property name="geometry">
     <rect>
      <x>700</x>
      <y>30</y>
      <width>104</width>
      <height>26</height>
     </rect>
    </property>
    <item>
     <property name="text">
      <string>sigmoid</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>softmax</string>
     </property>
    </item>
    <item>
     <property name="text">
      <string>linear</string>
     </property>
    </item>
   </widget>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>892</width>
     <height>24</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu">
    <property name="title">
     <string>메뉴</string>
    </property>
    <addaction name="actiontest_menu"/>
   </widget>
   <addaction name="menu"/>
  </widget>
  <action name="actiontest_menu">
   <property name="text">
    <string>test menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
